File: development/local-workspace/runtime-paths.mjs
import { resolve } from "node:path";

export const workspaceRoot = resolve(import.meta.dirname, "../..");
export const varRoot = resolve(process.env.PYTORCH_FIT_VAR_ROOT || resolve(workspaceRoot, "var"));
export const artifactRoot = resolve(
  process.env.PYTORCH_FIT_ARTIFACT_ROOT || resolve(workspaceRoot, "out"),
);

export function runtimePath(...parts) {
  return resolve(varRoot, ...parts);
}

export function artifactPath(...parts) {
  return resolve(artifactRoot, ...parts);
}

File: legacy/python/resume_builder/core/runtime_paths.py
"""Repository-local runtime and review-artifact path policy."""

from __future__ import annotations

import os
from pathlib import Path

REPO_ROOT = Path(__file__).resolve().parents[4]
VAR_ROOT = Path(os.environ.get("PYTORCH_FIT_VAR_ROOT", REPO_ROOT / "var")).resolve()
ARTIFACT_ROOT = Path(
    os.environ.get("PYTORCH_FIT_ARTIFACT_ROOT", REPO_ROOT / "out")
).resolve()


def var_path(*parts: str) -> Path:
    return VAR_ROOT.joinpath(*parts)


def artifact_path(*parts: str) -> Path:
    return ARTIFACT_ROOT.joinpath(*parts)
